src/model/Dough.java

```diff
@@ -1,9 +1,69 @@
 package model;
 
+import java.util.ArrayList;
+
 public class Dough extends Ingredient{
+    private ArrayList<Ingredient> ingredients;
+    private int freeSize = this.getSize();
     private enum DoughType {
         NEW_YORK,
         NEAPOLITAN,
         SICILIAN
     }
```

Comment:
move declaration after Dough class and make public
and make field with getter and setter of typeDoughType